### PR TITLE
php72-intl 7.2 (new formula)

### PR DESCRIPTION
**Pull request by romainneutron**
- [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/Homebrew/homebrew-php/pulls) for the same formula update/change?
- [x] Have you built your formula locally prior to submission with `brew install <formula>` (where `<formula>` is the name of the formula you're submitting)?
- [x] Does your submission pass `brew audit --strict --online <formula>` (after doing `brew install <formula>`)?

### CODE REVIEW

**Comment by javian** [comment]
@romainneutron can you fix
```
Error: 2 problems in 1 formula
homebrew/php/php72-intl:
  * Dependency 'icu4c' may be unnecessary as it is provided by macOS; try to build this formula without it.
```
and try to build this without the dependency ?

File: Formula/php72-intl.rb [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+class Php72Intl < AbstractPhp72Extension
+  init
+  desc "Wrapper for the ICU library"
+  homepage "http://php.net/manual/en/book.intl.php"
+
+  url PHP_SRC_TARBALL
+  sha256 PHP_CHECKSUM[:sha256]
+  version PHP_VERSION
+
+  depends_on "icu4c"
```
Comment (javian) [inline Formula/php72-intl.rb]: please remove dependency and see if it builds

**Comment by romainneutron** [comment]
PR updated, comment addressed

**Comment by cleptric** [comment]
Any news?

**Comment by javian** [comment]
@BrewTestBot test this please

**Comment by javian** [comment]
Thanks @romainneutron !

**Pull request closed without merging**